B827EBFFFEDDF5E8.json

```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "B827EBFFFEDDF5E8", 
+		"servers": [ 
+			{ 
+				"server_address": "ttn-router-asia-se", 
```

Comment:
This is not the correct URL for the router in Asia.

Reply:
I have updated it. Anyway I am also new to Git. Can you see my recent edits? Thanks